README.md

```diff
@@ -21,16 +21,14 @@ You can find the deployed project at [thetrashpanda.com](https://thetrashpanda.c
 <br>
 <br>
 
-🚫 Optional examples of using images with links for your tech stack, make sure to change these to fit your project
-
 ![MIT](https://img.shields.io/packagist/l/doctrine/orm.svg)
+![graphql](https://img.shields.io/badge/graphql-14.6.0-ff69b4)
 ![React](https://img.shields.io/badge/React-16.12.0-blue)
-![Typescript](https://img.shields.io/npm/types/typescript.svg?style=flat)
-[![Netlify Status](https://api.netlify.com/api/v1/badges/b5c4db1c-b10d-42c3-b157-3746edd9e81d/deploy-status)](netlify link goes in these parenthesis)
+![Apollo](https://img.shields.io/badge/Apollo-2.6-9cf)
+![styled-components](https://img.shields.io/badge/styled--components-v5.0.0-ff69b4)
```

Comment:
I would also add in testing with @apollo/react-testing, @testing-library/react, and specifically mention that we used jest WITHOUT express